apps/admin-gui-e2e/src/e2e/crawler.cy.js

```diff
@@ -0,0 +1,67 @@
+/// <reference types="cypress" />
+
+context('Actions', () => {
+
+  before(() => {
+    cy.login('', 'perun');
+  });
+
+  beforeEach(() => {
+    cy.visit('home')
+      .get('[data-cy=admin-button]')
+      .click();
+  });
+  // watch the video "Crawl Local Pages Using Cypress"
+// https://youtu.be/FDNeiwKWdb4
+  it('crawls all local pages', () => {
+    // let's keep track of all URLs we've visited
+    // NOTE: these URLs do not disambiguate between pages
+    // so if /about.html and /about are the same page,
+    // we will visit it twice if there are links to it
+    const visited = new Set()
+    const toVisit = ['/' , '/admin', '/myProfile']
```

Comment:
Shouldn't we check '/facilities' and '/organizations' as well?

Reply:
I omitted those because this implementation wouldn't find any new links on them anyway. 
This crawler is not intended for merge, just for people to see how it could be implemented. The problem is that our Angular app mostly uses onClick events to route, whereas this only looks for anchor elements. I couldn't figure out how to reliably get all the onClick event elements and retrieve the links from them.